Pear/Controller/FeedbackViewController.swift

```diff
@@ -310,3 +387,11 @@ extension FeedbackViewController: UICollectionViewDelegateFlowLayout {
     }
 
 }
+
+private class FinishButton: UIButton {
```

Comment:
I'm not sure if we should call the class a more generalized name as it seems this functionality is repeated for other types of buttons too.

Reply:
Sure! I think currently older buttons didn't use this approach but I can update them in another PR :)